src/sources/util/grpc/mod.rs

```diff
@@ -22,14 +23,19 @@ use tracing::Span;
 mod decompression;
 pub use self::decompression::{DecompressionAndMetrics, DecompressionAndMetricsLayer};
 
+mod connectionlimit;
```

Comment:
Nit: module names usually use `_` for delimiting multiple words so this would be `connection_limit`.